.github/workflows/test.yml

```diff
@@ -120,6 +131,8 @@ jobs:
           esac
           make -j4
           make install_sw
+          # Disable Bash debugging option.
+          set +x
```

Comment:
Is this necessary? I was under the impression that the shell process would exit after this and the following steps would be run on a new separate shell.

`set -x` while compiling OpenSSL seems useful.

Reply:
Good point! I was just adding the `set +x` as a pair of the `set -x`. For example, if you allocate a memory by `Xalloc`, you may want to add the `free` on the same layer. However as you mentioned, I also assume that the each shell process executed as an independent shell process. So, maybe the `set +x` is meaningless. I am going to remove the `set +x` command and rebase this PR after checking the behavior on my forked branch just in case.

Reply:
I confirmed that the `set -x` in the step doesn't affects to the later steps by the CI log below on my forked repository. I am rebasing this PR with removing the `set +x` command.
https://github.com/junaruga/openssl/actions/runs/5799098602/job/15718883212#step:4:5